src/patterns/gen-ai/aws-langchain-common-layer/index.ts

```diff
@@ -38,6 +38,14 @@ export interface LangchainLayerProps{
    * @default - none
    */
   readonly autoUpgrade?: boolean;
+  /**
+   * Optional: Local compute will be used when installing requirements.txt.
+   * By default if this is not true, a docker container will be spun up to install requirements.
+   * The string value will be the python alias used to install requirements.
+   *
+   * @default - false
```

Comment:
I was incorrect at the use, and had inadvertently marked this to force to boolean, it was the `autoUpgrade`:
```suggestion
   * @default - none
```